LayoutKit/Views/ReloadableViewLayoutAdapter+UICollectionView.swift

```diff
@@ -12,15 +12,15 @@ import UIKit
 
 extension ReloadableViewLayoutAdapter: UICollectionViewDelegateFlowLayout {
```

Comment:
Please add a doc comment to each extension that says overrides to these methods must call super.